FAQs/How to Model Alerts in OCSF.md

```diff
@@ -0,0 +1,32 @@
+# How to Model Alerts with OCSF - DRAFT
+
+In OCSF, an alertable signal is an event with the `is_alert` attribute set to `true`.  This is a newer attribute, starting with OCSF version 1.4.  The intent of `is_alert = true` is to signal that the event may require immediate attention by its consumer, usually a SIEM system or the product’s management console where an analyst can be notified, tickets created or the events prioritized.
+
+Not all OCSF events are alertable signals, and as of this writing only the `Detection Finding`, `Data Security Finding` event classes and the `Security Control` profile carry the `is_alert` boolean attribute.  Note the `Security Control` profile may be applied to many activity oriented classes, as well as the two aforementioned `Finding` classes, hence when applied any instances of these classes can be alertable signals.
```

Comment:
I do not see the `is_alert` attribute on `Detection Finding` or `Data Security Finding` classes, nor the `Security Control` profile.

Reply:
The latest merges have not yet been posted to the public server.  Check with your own copy after doing a local merge.